docs/docs/networking.md

```diff
@@ -21,7 +21,7 @@ VIPs map traffic from a single virtual address to multiple IP addresses and port
 Several [command-line flags](command-line-flags.html) determine Marathon's behavior with respect to networking.
 
 * `default_network_name` is injected as the `name` of a `container` mode network when left blank by an application.
-* `local_port_min` and `local_port_max` define a port range from which Marathon automatically allocates *service-port*s.
+* `local_port_min` and `local_port_max` define a port range from which Marathon automatically allocates \*service-port\*s.
```

Comment:
curious, why the backslashes here?

Reply:
if you don't escape the asterisks, the whole text looks like it's in italics in the text editor.